README.rst

```diff
@@ -51,10 +51,10 @@ icepyx is both a software library and a community composed of ICESat-2 data user
 
 icepyx aims to provide a clearinghouse for code, functionality to improve interoperability, documentation, examples, and educational resources that tackle disciplinary research questions while minimizing the amount of repeated effort across groups utilizing similar datasets. icepyx also hopes to foster collaboration, open-science, and reproducible workflows by integrating and sharing resources.
 
-Many of these tools began as Jupyter Notebooks developed for and during the cryosphere themed ICESat-2 Hackweek
-at the University of Washington in June 2019 or as scripts written and used by the ICESat-2 Science Team members.
-This project combines and generalizes these scripts into a unified framework, making them accessible for everyone.
-
+Many of the underlying tools from which icepyx was developed began as Jupyter Notebooks developed for and during the cryosphere-themed ICESat-2 Hackweek at the University of Washington in June 2019 or as scripts written and used by the ICESat-2 Science Team members. 
+Originally called icesat2py, the project combined and generalized these scripts into a unified framework, adding examples, documentation, and testing where necessary and making them accessible for everyone. 
+icepyx is now a domain-agnostic, standalone software package and community (under the broader `icesat2py GitHub organization <https://github.com/icesat2py>`_) that continues to build functionality - complete with examples, documentation, and testing - for obtaining and working with ICESat-2 data products locally and in the cloud. 
```

Comment:
```suggestion
icepyx is now a domain-agnostic, standalone software package and community (under the broader `icesat2py GitHub organization <https://github.com/icesat2py>`_) that continues to build functionality for obtaining and working with ICESat-2 data products locally and in the cloud. 
```
Small nitpick, but since the "examples, documentation, and testing" bit is already mentioned on Line 55, it's probably not needed here. Otherwise these lines look good.